content/en/getting-started/cameras.md

```diff
@@ -38,14 +38,12 @@ Here are a few examples from the community that are already being used by our us
-| 4           | 1.5        | ![img](/camera-comp/images/mortek_mc920.png) | MOERTEK    | MC920 1080P HD  | 8 - 18      | 85      | Nearly none | No           | Auto     | Yes       | very solid / looks like Logitech C920 / high quality packaging / own homepage / big company in china / very cheap / no need for distortion.json            |                                                                                      | purchase recommendation                     |
-| 3           | 2.5        | ![img](/camera-comp/images/ahnunva_c5.png)   | Ahnunva C5 | HD 2K 1080P     | 23          | 130     | little      | little       | Manual   | No        | tiny + small / solid worked                                                                                                                                | needs >40cm distance from board / problems with manual focus / needs distortion.json |                                             |
-| 2           | 3.5        | ![img](/camera-comp/images/anke_power.png)   | Anke Power | 1080P HD        | 16          | 110     | Yes         | No           | Auto     | Yes       | massive + stable                                                                                                                                           | still too mutch distortion / too expensive / needs distortion.json                   | no purchase recommendation                  |
-| 1           | 4          | ![img](/camera-comp/images/irarucw_2k.png)   | IRARUCW    | 2K HD           | 16          | 120     | Yes         | No           | Auto     | Yes       |                                                                                                                                                            | needs >45cm distance from board                                                      |                                             |
-| 1           | 5          | ![img](/camera-comp/images/aode_1080.png)    | AODE       | 1080 HD USB 2.0 | 10          | 90      | Yes         | Yes          | Auto     | Yes       | massive + stable                                                                                                                                           | still too mutch distortion / too expensive / USB problems (defect)                   | no purchase recommendation                  |
+| ![img](/camera-comp/images/elp_otg.png)      | ELP OTG HD 5mp        | 100     | None        | None         | Manual   | No        |
+| ![img](/camera-comp/images/mortek_mc920.png) | MOERTEK    | 85      | Nearly none | No           | Auto     | Yes       |
+| ![img](/camera-comp/images/HBV.png)   | HBVCAM-W2017-V22    | 110     | Little         | No           | Auto     |No       |                                             |
```

Comment:
I am a bit confused with the images here. The image called `HBV.png` is not the HBVCAM-W2017-V22, the OV9732 below is. I would just remove the `HBV.png` image. That camera will not work because it has the wrong lens on it.

Reply:
I will remove the HBV, but then we should also remove it from the guidelines (Discord / Marco)